front/src/applications/stdcmV2/views/StdcmViewV2.tsx

```diff
@@ -0,0 +1,85 @@
+
+import { Button } from '@osrd-project/ui-core';
+import { Location, ArrowUp, ArrowDown } from '@osrd-project/ui-icons';
+import { useSelector } from 'react-redux';
+
+import STDCM_REQUEST_STATUS from 'applications/stdcm/consts';
+import useStdcm from 'applications/stdcm/hooks/useStdcm';
+import { useOsrdConfSelectors } from 'common/osrdContext';
+import ScenarioExplorer from 'modules/scenario/components/ScenarioExplorer';
+import { Map } from 'modules/trainschedule/components/ManageTrainSchedule';
+
+import StdcmConsist from '../components/StdcmConsist';
+import StdcmDefaultCard from '../components/StdcmDefaultCard';
+import StdcmDestination from '../components/StdcmDestination';
+import StdcmHeader from '../components/StdcmHeader';
+import StdcmLoader from '../components/StdcmLoader';
+import StdcmOrigin from '../components/StdcmOrigin';
+
+// TODO
+// - translate labels
+
+const StdcmViewV2 = () => {
+  const { getProjectID, getScenarioID, getStudyID } = useOsrdConfSelectors();
+  const studyID = useSelector(getStudyID);
+  const projectID = useSelector(getProjectID);
+  const scenarioID = useSelector(getScenarioID);
+  const {
+    launchStdcmRequestV2,
+    cancelStdcmRequest,
+    setCurrentStdcmRequestStatus,
+    currentStdcmRequestStatus,
+  } = useStdcm();
+  const isPending = currentStdcmRequestStatus === STDCM_REQUEST_STATUS.pending;
+  const loaderRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    if (isPending) {
+      loaderRef?.current?.scrollIntoView({ behavior: 'smooth' });
+    }
+  }, [isPending]);
+
+  return (
+    <div className="stdcm-v2">
+      <StdcmHeader />
+      <div className="stdcm-v2__body">
+        <div className="stdcm-v2-simulation-settings">
+          <div>
+            <div className="mb-4">
+              <ScenarioExplorer
+                globalProjectId={projectID}
+                globalStudyId={studyID}
+                globalScenarioId={scenarioID}
+              />
+            </div>
+            <StdcmConsist isPending={isPending} />
+          </div>
+          <div className="stdcm-v2__separator" />
+          <div className="stdcm-v2-simulation-itinirary">
+            {/* //TODO: rename StdcmDefaultCard */}
+            {/* <StdcmDefaultCard text="Indiquer le sillon antérieur" Icon={<ArrowUp size="lg" />} /> */}
+            <StdcmOrigin isPending={isPending} />
+            {/* <StdcmDefaultCard text="Ajouter un passage" Icon={<Location size="lg" />} /> */}
+            <StdcmDestination isPending={isPending} />
+            {/* <StdcmDefaultCard text="Indiquer le sillon postérieur" Icon={<ArrowDown size="lg" />} /> */}
+            <Button
+              label="Obtenir la simulation"
+              onClick={() => {
+                setCurrentStdcmRequestStatus(STDCM_REQUEST_STATUS.pending);
+                launchStdcmRequestV2();
+              }}
+            />
+            {isPending && <StdcmLoader cancelStdcmRequest={cancelStdcmRequest} ref={loaderRef} />}
```

Comment:
Here is what I see on your PR : 
![Capture d’écran 2024-05-23 à 15 13 06](https://github.com/OpenRailAssociation/osrd/assets/113104137/3db223d7-1a6d-4d7c-b79e-d517e243b0ea)

On the sketch model : 
![Capture d’écran 2024-05-23 à 15 14 59](https://github.com/OpenRailAssociation/osrd/assets/113104137/25311c3d-ba5e-4258-af3e-dd989f28265c)

It seems that the text should be center and in bold. Also It seems that here is still this kind of teal inline border at the top of your button which doesn't seem to be there in the model.